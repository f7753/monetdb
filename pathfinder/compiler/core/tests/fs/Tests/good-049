#!/bin/bash

pf -s9 <<'EOT'
processing-instruction foo { "bar" }
EOT
echo $?
