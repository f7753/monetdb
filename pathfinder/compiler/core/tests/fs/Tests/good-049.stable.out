stdout of test 'good-049` in directory 'compiler/core/tests/fs` itself:


# 15:46:21 >  
# 15:46:21 >  Mtimeout -timeout 60 ./good-049 good-049 
# 15:46:21 >  

0

# 15:46:21 >  
# 15:46:21 >  Done.
# 15:46:21 >  

